plugins/out_s3/s3.c

```diff
@@ -2363,6 +2353,15 @@ static struct flb_config_map config_map[] = {
      " data will be locally buffered at any given point in time."
     },
 
+    {
```

Comment:
Add a warning if normal limit size is configured and note that this is the setting to use